src/main/java/com/yelp/nrtsearch/tools/nrt_utils/incremental/DeleteIncrementalSnapshotsCommand.java

```diff
@@ -69,12 +69,13 @@ public class DeleteIncrementalSnapshotsCommand implements Callable<Integer> {
 
   @CommandLine.Option(
       names = {"-c", "--credsFile"},
-      description = "File holding AWS credentials, uses default locations if not set")
+      description =
+          "File holding AWS credentials, uses default locations if not set; Will use DefaultCredentialProvider if this is unset.")
```

Comment:
Should the description contain only `uses default credential provider chain if this is unset` instead of also including `uses default locations if not set`?